bluebutton-data-pipeline-fhir-load/src/test/java/gov/hhs/cms/bluebutton/datapipeline/fhir/transform/DataTransformerTest.java

```diff
@@ -824,6 +1036,22 @@ private static void assertAdjudicationEquals(String expectedCategoryCode, BigDec
 	}
 
 	/**
+	 * @param expectedCategoryCode
+	 *            the expected {@link Coding#getCode()} of the
+	 *            {@link ItemAdjudicationComponent#getCategory()} to verify is
+	 *            not present
+	 * @param actuals
+	 *            the actual {@link ItemAdjudicationComponent}s to verify
+	 */
+	private static void assertAdjudicationNotPresent(String expectedCategoryCode,
+			List<ItemAdjudicationComponent> actuals) {
+		Optional<ItemAdjudicationComponent> adjudication = actuals.stream()
+				.filter(a -> DataTransformer.CODING_SYSTEM_ADJUDICATION_CMS.equals(a.getCategory().getSystem()))
+				.filter(a -> expectedCategoryCode.equals(a.getCategory().getCode())).findAny();
+		Assert.assertFalse(adjudication.isPresent());
+	}
```

Comment:
Are you finding these stream-leveraging assertions to be readable enough? I do most of the time, but I'm curious if other folks do, too.